docs/nowdns.md

```diff
@@ -0,0 +1,26 @@
+# Now-DNS
+
+## Configuration
+
+### Example
+
+```json
+{
+  "settings": [
+    {
+      "provider": "nowdns",
+      "domain": "domain.com",
+      "host": "@",
+      "username": "username",
+      "password": "password",
+    }
+  ]
+}
+```
+
+### Compulsory parameters
+
+- `"domain"` your domain name
+- `"host"` leave it set to `"@"`
```

Comment:
If the host can only be `"@"`, should we just remove the option then?

Reply:
I did that and it was failing to update the record.